netbeans.apache.org/src/content/tutorials/index.asciidoc

```diff
@@ -25,81 +25,81 @@
-- link:nbm-workflow.html[NetBeans Platform Workflow Tutorial]
-- link:nbm-projectsamples.html[NetBeans Project Sample Module Tutorial]
-- link:nbm-projectextension.html[NetBeans Project Type Extension Module Tutorial]
-- link:nbm-projecttype.html[NetBeans Project Type Module Tutorial]
-- link:nbm-property-editors.html[NetBeans Property Editor Tutorial]
-- link:nbm-selection-2.html[NetBeans Selection Management Tutorial II—Using Nodes]
-- link:nbm-selection-1.html[NetBeans Selection Management Tutorial I—Using a TopComponent&#39;s Lookup]
-- link:nbm-server-plugin.html[NetBeans Server-Skeleton Module Tutorial]
-- link:nbm-nodesapi.html[NetBeans System Properties Module Tutorial]
-- link:nbm-visual_library2.html[NetBeans Visual Database Explorer Tutorial]
-- link:nbm-visual_library.html[NetBeans Visual Library Tutorial]
-- link:nbm-visual_library3.html[NetBeans Visual Library Tutorial for Java Applications]
-- link:nbm-quick-start-visual.html[NetBeans Visual Library Tutorial for Standard Java SE Applications]
-- link:nbm-wizard.html[NetBeans Wizard Module Tutorial]
-- link:nbm-xmleditor.html[NetBeans XML Editor Extension Module Tutorial]
-- link:nbm-feedreader_background.html[Preparing to Create the Feed Reader Application]
-- link:nbm-quick-search.html[Quick Search Integration Tutorial]
-- link:section.html[]
-- link:nbm-10-top-apis.html[Top 10 NetBeans APIs]
-- link:nbm-povray-1.html[Writing POV-Ray Support for NetBeans I—Application Structure &amp;amp; File Support]
-- link:nbm-povray-3.html[Writing POV-Ray Support for NetBeans III—Implementing a Project Type]
-- link:nbm-povray-2.html[Writing POV-Ray Support for NetBeans II—Project Type Design]
-- link:nbm-povray-4.html[Writing POV-Ray Support for NetBeans IV—Providing Project Templates]
-- link:nbm-povray-9.html[Writing POV-Ray Support for NetBeans IX—Build Support]
-- link:nbm-povray-5.html[Writing POV-Ray Support for NetBeans V—Creating an API]
-- link:nbm-povray-8.html[Writing POV-Ray Support for NetBeans VIII—Implementing ViewService and its Actions]
-- link:nbm-povray-6.html[Writing POV-Ray Support for NetBeans VI—Implementing the API]
-- link:nbm-povray-7.html[Writing POV-Ray Support for NetBeans VII—Support For Running POV-Ray]
-- link:nbm-povray-10.html[Writing POV-Ray Support for NetBeans X—Conclusion]
+- link:nbm-glossary.asciidoc[Basic Terms for NetBeans Rich Client Application Development]
+- link:nbm-code-generator.asciidoc[Code Generator Integration Tutorial]
+- link:nbm-visual_library4.asciidoc[DRAFT: NetBeans Visual Library Widget Creation Tutorial]
+- link:nbm-filetype.asciidoc[File Type Integration Tutorial]
+- link:nbm-propertyeditors-integration.asciidoc[Integrated Property Editors in NetBeans]
+- link:nbm-javacc-lexer.asciidoc[JavaCC Lexer Generator Integration Tutorial for the NetBeans Platform]
+- link:nbm-javacc-parser.asciidoc[JavaCC Parser Generator Integration Tutorial for the NetBeans Platform]
+- link:index.asciidoc[Moved!]
```

Comment:
I suggested removing this link in a previous PR. It just isn't doing anything.

Reply:
Removed linkLindex.asciidoc [Moved!] line